deCODE_to_MT.py

```diff
@@ -90,8 +89,17 @@ def main(
         )
         | (hl.len(mt.alleles) != 2)
         | ((hl.len(mt.alleles) == 2) & (mt.n_unsplit_alleles != 2))
-        | (hl.is_missing(mt["InbreedingCoeff"]))
-        | ((hl.is_defined(mt["InbreedingCoeff"])) & (mt["InbreedingCoeff"] < -0.3))
+    )
+    mt = mt.filter_rows(filter_conditions, keep=False)
+
+    # Exclude variants with inbreeding coefficient < -0.3
+    # InbreedingCoeff was calculated by function 'bi_allelic_site_inbreeding_expr'
+    mt = hl.variant_qc(mt)
```

Comment:
Is calling `variant_qc` here necessary?

Reply:
I am not sure whether I should call `variant_qc` for the `gnomad.utils.annotations.bi_allelic_site_inbreeding_expr` function. Just have a double check, I guess it this function needs any input parameters, it should call them itself. I will delete this sentence. Thank you!

Reply:
That's right -- `bi_allelic_site_inbreeding_expr(mt.GT)` means that `bi_allelic_site_inbreeding_expr` only has access to the `GT` field, so there's no way for the function to access anything else.

Reply:
I have committed these changes! Thanks @lgruen!